sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_spons_tiles_v1/query.sql

```diff
@@ -0,0 +1,184 @@
+WITH combined AS (
+  SELECT
+    context_id,
+    DATE(submission_timestamp) AS submission_date,
+    'desktop' AS form_factor,
+    normalized_country_code AS country,
+    advertiser,
+    SPLIT(metadata.user_agent.os, ' ')[SAFE_OFFSET(0)] AS normalized_os,
+    release_channel,
+    position,
+    CASE
+    WHEN
+      reporting_url IS NULL
+    THEN
+      'remote settings'
+    ELSE
+      'contile'
+    END
+    AS provider,
```

Comment:
```suggestion
    IF(reporting_url IS NULL, 'remote settings', 'contile') AS provider,
```

what do you think about switching these simple case statements to IF functions? My thinking is that it would help with the readability of the select statements.